app/assets/stylesheets/text-extraction.scss

```
@@ -0,0 +1,53 @@
+.language-pill {
+  gap: 0.35rem;
+  display: inline-flex;
+  align-items: center;
+  padding-left: 0.55rem;
+  padding-right: 0.45rem;
+
+  .language-label {
+    border-inline-end: var(--bs-border-width) var(--bs-border-style) var(--bs-border-color);
+    font-size: 0.875em;
+    line-height: 1.45em;
+    padding-block: 0.1rem;
+    padding-inline-end: 0.45rem;
+    text-wrap: nowrap;
+  }
+
+  .pill-close {
+    font-size: .6rem;
+  }
+}
+
+.dropdown-content {
+  background-color: #f1f1f1;
+  overflow-y: auto;
+  max-height: 180px;
+  label {
+    padding: 5px 10px;
+    cursor: pointer;
+  }
+
+  label:hover {
+    background-color: #ddd;
+  }
+}
+
+.text-cardinal {
+  color: $color-cardinal-red;
```

```suggestion
  color: var(--bs-primary);
```

After this change is made, can this just be a `css` file (e.g. not a scss)?

---

Made the code change, though every other file in the stylesheets folder is .scss, so not sure if changing the type of file makes sense (for the sake of consistency)